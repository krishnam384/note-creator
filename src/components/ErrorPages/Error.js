import React from "react";
import PropTypes from "prop-types";

export default function Error({ error }) {
  return <small style={{ color: "red" }}>{error}</small>;
}

Error.propTypes = {
  error: PropTypes.string,
};
