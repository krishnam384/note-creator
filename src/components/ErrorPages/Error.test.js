import React from "react";
import { shallow } from "enzyme";
import Error from "./Error";

/* eslint no-undef: 0 */

describe("Testing Error Component", () => {
  it("Snapshot Testing of Error Component", () => {
    const wrapper = shallow(<Error />);

    expect(wrapper).toMatchSnapshot();
  });

  // it("Snapshot Testing of Error Component", () => {
  //   const wrapper = shallow(<Error error='Page Not Found' />);

  //   expect(wrapper.prop("error")).toEqual("Page Not Found");
  // });
});
