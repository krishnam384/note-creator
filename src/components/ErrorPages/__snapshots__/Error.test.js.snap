// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Error Component Snapshot Testing of Error Component 1`] = `
<small
  style={
    Object {
      "color": "red",
    }
  }
>
  *
</small>
`;
